conda/recipes/libraft/build_libraft.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
-# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+# Copyright (c) 2022-2024, NVIDIA CORPORATION.
 
-./build.sh libraft --allgpuarch --compile-lib --build-metrics=compile_lib --incl-cache-stats --no-nvtx
+./build.sh libraft --allgpuarch --compile-lib --build-metrics=compile_lib --incl-cache-stats --no-nvtx -n
+cmake --install cpp/build --component compiled
```

Comment:
This now looks a lot like https://github.com/rapidsai/raft/blob/branch-24.04/conda/recipes/libraft/build_libraft_static.sh except with a different component being installed.